### ISSUE TITLE
Game lags severely in basalt deltas biome

### ISSUE THREAD

**Issue by Yum-Muy** [wVibzz/ImmersiveWinds#34]
With the mod installed, it drops down to around 30 and below, but without it, it usually sits above 60. Below is my current modlist.

![Screenshot 2024-12-03 211329](https://github.com/user-attachments/assets/b7f2b2ca-b011-4de0-b019-9d5c568324f8)
![Screenshot 2024-12-03 211322](https://github.com/user-attachments/assets/c4753f4d-17c5-4ef9-baab-19609a92acc4)
![Screenshot 2024-12-03 211318](https://github.com/user-attachments/assets/80840510-6cf3-4372-98d2-7554de5f3016)
![Screenshot 2024-12-03 211313](https://github.com/user-attachments/assets/d4d38e30-0ccf-4996-8aee-72213b9a9301)
![Screenshot 2024-12-03 211304](https://github.com/user-attachments/assets/3e705968-56ad-4dd9-930c-4083d4b79e87)
![Screenshot 2024-12-03 211254](https://github.com/user-attachments/assets/9a2c2e01-f003-427b-9c8d-1598c64eaff7)
![Screenshot 2024-12-03 211245](https://github.com/user-attachments/assets/58377464-2149-4718-81ca-1324aa51227c)
![Screenshot 2024-12-03 211229](https://github.com/user-attachments/assets/d2982aab-13a9-4153-be05-f176277d40fe)

**Comment by wVibzz** [comment]
so you tested with and without my mod specifically?

**Comment by Yum-Muy** [comment]
Yup yup, and here is a ss with only the mod and its dependencies installed (still really laggy):
![2024-12-05_19 49 37](https://github.com/user-attachments/assets/81d250df-c3f7-4529-b603-aafef7838c73)

**Comment by wVibzz** [comment]
Ok im looking into it, and should be fixed within the next update.

**Comment by wVibzz** [comment]
I just looked at your SS and it says minecraft only has 4gb of ram. You should def increase the amount of allocated ram especially with the amount of mods you run. Seeming how only 46% is allocated. I would do up to 85 or 90% of ram of your 8 gigs and see if your performance improves

**Comment by Yum-Muy** [comment]
if you dont mind me asking, how do i allocate more vram? cause those 8 gigs are from my gpu lol
i recently updated my normal ram frmo 4 to about 6 gb, but gonna be honest i dont feel any difference 😅

**Comment by wVibzz** [comment]
Plenty of tutorials and what not online. Find optimizations that work for you. It could just be your low specs. However when i get home ill run one more test that i forgot to do and see if i get bad performance. Otherwise it could just be your hardware.

**Comment by Yum-Muy** [comment]
gotcha, and thank you!

**Comment by wVibzz** [comment]
Coming back to this issue, there was lag in this biome because of the calculations my mod was doing for every particle there. This is fixed, and will release with next update.

**Comment by Yum-Muy** [comment]
Awesome, cant wait to try it out soon!!!!!!
